service/worker/deployment/deployment_workflow.go

```diff
@@ -59,7 +59,8 @@ var (
 		},
 	}
 
-	errAlreadyExists = errors.New("Task queue already exists in deployment")
+	errAlreadyExists             = errors.New("task queue already exists in deployment")
+	ErrMaxTaskQueuesInDeployment = errors.New("maximum number of task queues registed in deployment")
```

Comment:
```suggestion
	ErrMaxTaskQueuesInDeployment = errors.New("maximum number of task queues registered in deployment has reached")
```

Reply:
ah, noticed the typo - I think the following is even better:
`maximum number of task queues have been registered in deployment`